MonoGame.Framework/Graphics/SpriteFont.cs

```diff
@@ -83,7 +106,7 @@ public int GetHashCode(char b)
         /// <remarks>Can be used to calculate character bounds when implementing custom SpriteFont rendering.</remarks>
         public Dictionary<char, Glyph> GetGlyphs()
```

Comment:
> For now I could generate the Dictionary<char, Glyph> on the fly 

That is not a bad idea... make it a cost at request time for only people that use it.